nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/pubsub/AbstractPubSubBroker.java

```diff
@@ -75,29 +72,26 @@ public void initialize(String topic, String discriminator) {
                         + (char) DISCRIMINATOR_SEP + "': " + discriminator);
             }
         }
-        bufferedInvalidations = newInvalidations();
         PubSubService pubSubService = Framework.getService(PubSubService.class);
         pubSubService.registerSubscriber(topic, this::subscriber);
     }
 
     /**
-     * Closes this invalidator and releases resources.
+     * Closes this broker and releases resources.
      */
     public void close() {
         PubSubService pubSubService = Framework.getService(PubSubService.class);
         pubSubService.unregisterSubscriber(topic, this::subscriber);
-        // not null to avoid crashing subscriber thread still in flight
-        bufferedInvalidations = newInvalidations();
     }
 
     protected static final byte DISCRIMINATOR_SEP = ':';
 
     /**
-     * Sends invalidations to other nodes.
+     * Sends a message to other nodes.
      */
```

Comment:
since 9.3?

Reply:
The whole class is effectively new code and the class already has `@since 9.3`